tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/Generators/ReadMeGeneratorTool.cs

```diff
@@ -0,0 +1,215 @@
+                engPathOption,
+                modelOption,
+                outputPathOption,
+                packagePathOption,
+                serviceDocumentationOption,
+                templatePathOption,
+            };
+
+            command.SetHandler(async (ic) => await HandleCommand(ic, ic.GetCancellationToken()));
+
+            return command;
+        }
+
+        public override async Task HandleCommand(InvocationContext ctx, CancellationToken ct)
+        {
+            var pr = ctx.ParseResult;
+
+            var templatePath = pr.GetValueForOption(templatePathOption);
+            var serviceDocumentation = pr.GetValueForOption(serviceDocumentationOption);
+            var packagePath = pr.GetValueForOption(packagePathOption);
+            var outputPath = pr.GetValueForOption(outputPathOption);
+            var engPath = pr.GetValueForOption(engPathOption);
+            var model = pr.GetValueForOption(modelOption);
+
+            var chatClient = openAiClient.GetChatClient(model);
+
+            await Generate(
+                chatClient: chatClient,
+                engPath: engPath,
+                templatePath: templatePath,
+                serviceDocumentation: new Uri(serviceDocumentation),
+                packagePath: packagePath,
+                outputPath: outputPath);
+        }
+
+        async Task Generate(ChatClient chatClient, string templatePath, Uri serviceDocumentation, string packagePath, string outputPath, string engPath)
+        {
+            ArgumentException.ThrowIfNullOrEmpty(templatePath);
+            ArgumentNullException.ThrowIfNull(serviceDocumentation);
+            ArgumentException.ThrowIfNullOrEmpty(outputPath);
+            ArgumentException.ThrowIfNullOrEmpty(engPath);
+
+            var readmeText = await File.ReadAllTextAsync(templatePath);
+
+            var messages = new ChatMessage[]{
+                new SystemChatMessage(
+                    string.Join("\n",
+                        "We're going to create some READMEs.",
+                        "The parameters are:",
+                        "* A URL that contains service documentation, to be used when creating key concepts, an introduction blurb and any other places where conceptual docs are needed",
+                        "* A package path that we can use to generate documentation links",
+                        "Here are some more rules to follow:",
+                        "- Do not touch the following sections, or its subsections: Contributing.",
+                        "- Do not generate sample code.",
+                        "- Rules for proper readmes can be found here: https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-TEMPLATE.md"
+                    )
+                ),
+                new SystemChatMessage("The readme template is this: \"" + readmeText + "\" which we fill in with the user parameters, which follow:"),
```

Comment:
```suggestion
                new SystemChatMessage($"The readme template is this: \"{readmeText}\" which we fill in with the user parameters, which follow:"),
```

No reason to use a different pattern here.  The bling string is just fine.